folium/raster_layers.py

```diff
@@ -222,6 +223,9 @@ def __init__(
             attribution=attr,
             **kwargs
         )
+        if cql_filter:
+            # special parameter that shouldn't be camelized
+            self.options["cql_filter"] = cql_filter
```

Comment:
Can we pop it from the kw args to avoid adding an extra arg entry? Not sure if this is best but maybe something like:

```python
cql_filter = kwargs.pop("cql_filter", None)
if cql_filter:
    # special parameter that shouldn't be camelized
    self.options["cql_filter"] = cql_filter
```